build.gradle

```diff
@@ -55,6 +55,7 @@ ext {
     // This value is always overwritten, but Gradle needs the variable to be initialized.
     localRepo = ".git"
 }
+
```

Comment:
Please remove any extraneous whitespace changes, like this one.  They make review more difficult.